packages/backend/src/Actions/deleteActivity.js

```diff
@@ -0,0 +1,8 @@
+export const DELETE_ACTIVITY = 'DIRECT_LINE/DELETE_ACTIVITY';
```

Comment:
Do you  need the standard "invoked" "handled" "failed" for all of these?

Reply:
I think today, only POST_ACTIVITY is async.

We need to redo the prefix "family" to better identify actions to/from Direct Line or to/from Web Chat.